src/js/components/Button/Button.js

```diff
@@ -23,7 +25,9 @@ const Button = ({
   children,
   className,
   icon,
+  iconColor,
```

Comment:
I can do that. I'll need to remove the hovering effect for icons with theme props in buttons then. Otherwise when you hover the icon the color changes but not when you hover the button.